modules/ml/src/test/java/org/apache/ignite/ml/naivebayes/compound/CompoundNaiveBayesTrainerTest.java

```diff
@@ -0,0 +1,87 @@
+package org.apache.ignite.ml.naivebayes.compound;
```

Comment:
Could you please add Apache header containing the license?

Reply:
I added the header.